core/templates/pages/contributor-dashboard-page/modal-templates/translation-modal.component.ts

```diff
@@ -178,9 +180,13 @@ export class TranslationModalComponent {
           }
           this.activeWrittenTranslation.html = '';
           this.uploadingTranslation = false;
+        }, () => {
+          this.contextService.resetImageSaveDestination();
```

Comment:
Why did we move this below? Just curious if OppiaAngularRoot is responsible for this change or not.